packages/nodejs/src/net/NodeJsUdpChannel.ts

```diff
@@ -23,6 +25,8 @@ import { NodeJsNetwork } from "./NodeJsNetwork.js";
 
 const logger = Logger.get("NodejsChannel");
 
+const UDP_SEND_TIMEOUT_CHECK_INTERVAL_MS = 1_000; // UDP should be send out in some ms so if we needed 1s+, we have a problem
```

Comment:
This is a pretty short timeout of DNS is involved in the send... I think that doesn't affect us currently but maybe this should be configurable?

Reply:
We usually send to IPs directly and not to names, so should not be an issue? I can also go higher